### ISSUE TITLE
Feature request: Add setting timezone to user preferences

### ISSUE THREAD

**Issue by DirkM20** [Snapmaker/Snapmaker2-Controller#198]
not everybody lives in UTC...

**Comment by voidrin** [comment]
Start watching this feature request, will pass to my colleague and consider putting into backlogs.

**Comment by darienm** [comment]
Agreed. There is a language setting, which could be renamed to "Locale" and also include the time zone selection or at the very least a positive or negative offset from UTC.
